### DISCUSSION TITLE
перестал  работать

### DISCUSSION THREAD

**Discussion by wizarvamp** [lirix360/ReadmangaGrabber#20] [General]
Подскажите до этого все работало. А сегодня везде пишет вот это 
![01](https://user-images.githubusercontent.com/90101234/133246645-32909077-9343-43fc-b712-62db8f6073e6.jpg)
и так слюбой мангой и на readmanga так же.
Версия 2.0 beta 3

**Comment by lirix360** [comment]
Исправлено в beta 4.
